test/integration/client_test.cpp

```diff
@@ -168,25 +171,24 @@ TEST_F(ClientServerTest, SendTxWhenInvalidJson) {
 }
 
 TEST_F(ClientServerTest, SendTxWhenStatelessInvalid) {
-  EXPECT_CALL(*svMock, validate(A<const iroha::model::Transaction &>()))
-      .WillOnce(Return(false));
-  auto shm_tx = shared_model::proto::TransactionBuilder()
-      .creatorAccountId("some@account")
-      .txCounter(1)
-      .createdTime(iroha::time::now())
-      .setAccountQuorum("some@account", 2)
-      .build()
-      .signAndAddSignature(
-          shared_model::crypto::CryptoProviderEd25519Sha3::
-          generateKeypair());
-  auto tx = *shm_tx.makeOldModel();
-
-  ASSERT_EQ(iroha_cli::CliClient(Ip, Port).sendTx(tx).answer,
+  // creating stateless invalid tx
+  auto shm_tx = TestTransactionBuilder()
+                    .creatorAccountId("some@account")
+                    .txCounter(1)
+                    .createdTime(iroha::time::now())
+                    .setAccountQuorum("some@@account", 2)
+                    .build();
+  auto *old_tx = shm_tx.makeOldModel();
+
+  ASSERT_EQ(iroha_cli::CliClient(Ip, Port).sendTx(*old_tx).answer,
             iroha_cli::CliClient::OK);
-  ASSERT_EQ(iroha_cli::CliClient(Ip, Port)
-                .getTxStatus(iroha::hash(tx).to_string())
-                .answer.tx_status(),
-            iroha::protocol::TxStatus::STATELESS_VALIDATION_FAILED);
+  auto tx_hash = shm_tx.hash();
+  ASSERT_EQ(
+      iroha_cli::CliClient(Ip, Port)
```

Comment:
Is it really need to open connection each time?

Reply:
That is how Torii works at the moment. Fixing that out of scope of the task(